pages/coinbase/signin.py

```diff
@@ -0,0 +1,28 @@
+from selenium import webdriver
+from selenium.webdriver.support import expected_conditions as EC  
+from selenium.webdriver.support.ui import WebDriverWait as wait
+from selenium.webdriver.common.by import By
+import secure.cookies
+import secure.creds
+import config
+import time
+
+
+class Sign_In_Page():
+
+    def __init__(self,driver, get_SignIn_url: bool=None):  #if the driver is starting at the sign in page use keyword argument True at instantiation 
```

Comment:
This parameter doesn't need to be an annotation; annotations are just metadata and we don't use it.

Just make this a bool.